### PR TITLE
feat: seasonal (pvp-season) mode, seasonal perks, and missing crafts

### PR DESCRIPTION
**Pull request by DysektAI**
## Summary

Adds the new **Seasonal Character** game mode (BSG EFT patch 1.1.0.0) to the overlay and the seasonal data tarkov.dev does not yet serve.

### Feature (commit 1 — `feat:`)
- **`pvp-season` as a first-class upstream mode.** tarkov.dev serves it (`json.tarkov.dev/pvp-season/*`, listed under `gameModes` at `/endpoints`), so `SUPPORTED_GAME_MODES` is now `regular` / `pve` / `pvp-season`, with a `src/overrides/modes/pvp-season/` section that merges like the other modes.
- **`seasonalPerks` addition (39 perks).** BSG Seasonal Character perks tarkov.dev has no endpoint for. Everything references tarkov.dev by **id** and resolves names via locale — `itemFilter` uses tarkov.dev's served `ItemFilters` shape (id arrays for `allowedItems`/`allowedCategories`/`excludedItems`/`excludedCategories`), `skillIds`/`traderIds`/`mailTemplateId` are tarkov.dev ids, and effect enums are canonical keys. Only perk `name`/`description` are inline English (no upstream source, like edition titles).
- **`craftsAdd` addition (23 crafts).** Hideout crafts missing from tarkov.dev (incl. the Black Division decryption chain), in tarkov.dev's served craft shape (`station`/`level`/`duration`/`requiredItems`/`productItem`, all ids). Quest-gated crafts carry `taskUnlock: null` as a fillable placeholder (the capture doesn't expose the unlocking task id). Christmas Tree event-station crafts are excluded (tarkov.dev tracks those separately).
- **Divergence registry scoped to `DIVERGENCE_MODES` (`regular`/`pve`)** — the mirror-detection concept is a regular↔pve artifact. PvE/divergence provenance relabeled as "verified in-game" (dropped the `eft-reference` source).
- Descriptive **`User-Agent`** on `json.tarkov.dev` fetches.
- New schemas (`seasonal-perk`, `craft-additions`), TS types, tests, and docs (`AGENTS.md`, `docs/INTEGRATION.md` consumption guide).

### Formatting (commit 2 — `style:`)
Repo-wide Prettier normalization (ts/json/json5/md). The `format`/`format:check` scripts were expanded to cover json/json5/md and a `.prettierignore` was added (excludes the generator-locked `storyChapters.json5`, which was re-canonicalized to its generator format). This commit is mechanical formatting only — review it separately from the feature commit.

## Testing
- `npm run typecheck` — clean
- `npm run validate` — all source files valid
- `npm run build` — builds `dist/overlay.json` (seasonalPerks: 39, craftsAdd: 23)
- `npm test` — **320/320 pass**
- `npm run format:check` — clean
- `npm run check-overrides` / `npm run eft:audit` — 0 actionable; PvE alignment 0 GAP/CONFLICT/STALE

## Known limitations / caveats
- **craftsAdd `taskUnlock`:** the 14 quest-gated crafts carry `taskUnlock: null` — they're quest-locked in-game but the `/season/` capture has no `quest_list`, so the unlocking task id can't be populated yet. Fine for recipe/ingredient lookups; progression tools should note it.
- **15 of 64 perk `skillIds`** aren't in tarkov.dev's `skills` list yet (mostly the `Usec*` faction skills, which tarkov.dev is missing while it has the Bear equivalents) — the ids are correct; names resolve once upstream adds them.
- Data fidelity relies on the local in-game capture being accurate; structure, shape, and id-resolvability were validated against live tarkov.dev.

<a href="https://cubic.dev/pr/tarkovtracker-org/tarkov-data-overlay/pull/257?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

___

## **CodeAnt-AI Description**
Add Seasonal Character data and correct game-mode quest information

### What Changed
- Adds `pvp-season` support, including 39 Seasonal Character perks and 23 missing hideout crafts.
- Updates quest corrections and divergence data for EFT 1.1.0.0, including revised experience, level, map, objective, and count values.
- Removes obsolete corrections that were overriding accurate upstream data and keeps seasonal corrections separate from regular/PvE wiki comparisons.
- Improves wiki comparison by recognizing maps mentioned in both links and text, handling stacked task-name suffixes, and distinguishing “found in raid” from “not found in raid.”
- Uses a descriptive client identity for tarkov.dev requests and selects the newest nested EFT quest reference file for audits.
- Adds validation and regression coverage for seasonal data, objective divergence counts, task-name matching, request headers, and overlay scoping.

### Impact
`✅ Seasonal Character perks available`
`✅ Missing hideout crafts available`
`✅ Correct quest rewards and objectives for EFT 1.1.0.0`
`✅ Fewer false wiki/API discrepancies`
`✅ Fewer blocked tarkov.dev requests`
<details><summary><strong>💡 Usage Guide</strong></summary>

### Checking Your Pull Request
Every time you make a pull request, our system automatically looks through it. We check for security issues, mistakes in how you're setting up your infrastructure, and common code problems. We do this to make sure your changes are solid and won't cause any trouble later.

### Talking to CodeAnt AI
Got a question or need a hand with something in your pull request? You can easily get in touch with CodeAnt AI right here. Just type the following in a comment on your pull request, and replace "Your question here" with whatever you want to ask:
<pre>
<code>@codeant-ai ask: Your question here</code>
</pre>
This lets you have a chat with CodeAnt AI about your pull request, making it easier to understand and improve your code.

#### Example
<pre>
<code>@codeant-ai ask: Can you suggest a safer alternative to storing this secret?</code>
</pre>

### Preserve Org Learnings with CodeAnt
You can record team preferences so CodeAnt AI applies them in future reviews. Reply directly to the specific CodeAnt AI suggestion (in the same thread) and replace "Your feedback here" with your input:
<pre>
<code>@codeant-ai: Your feedback here</code>
</pre>
This helps CodeAnt AI learn and adapt to your team's coding style and standards.

#### Example
<pre>
<code>@codeant-ai: Do not flag unused imports.</code>
</pre>

### Retrigger review
Ask CodeAnt AI to review the PR again, by typing:
<pre>
<code>@codeant-ai: review</code>
</pre>

### Check Your Repository Health
To analyze the health of your code repository, visit our dashboard at [https://app.codeant.ai](https://app.codeant.ai). This tool helps you identify potential issues and areas for improvement in your codebase, ensuring your repository maintains high standards of code health.

</details>

### CODE REVIEW

File: src/additions/seasonalPerks.json5 [hunk 1]
```diff
@@ -0,0 +1,1116 @@
...
+        skillIds: ['Crafting'],
+        intValue: 51,
+        effectId: 'skill_level_preset',
+      },
+      {
+        multiplicator: 0.87,
+        effectId: 'craft_time_multiplicator',
+      },
+    ],
+  },
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/additions/seasonalPerks.json5:74]: **Intentional — kept exactly as the game presents it.** `description` is the verbatim in-game display text; `effects[]` are the raw values the game actually applies. BSG's display strings and underlying values are not always consistent — confirmed for Handyman (text "-50%" vs `craft_time_multiplicator` 0.87), Polydipsia (text "15%" vs `hydration_drain_multiplicator` 1.2), and Average (text/`skill_level_preset` exclude Crafting while `skill_not_growing` includes it).

Both sides are captured faithfully and must **not** be reconciled — "fixing" either side would misrepresent the game. This is now documented in the file header and with inline `NB:` notes on each of the three perks (commit 6a55758). Resolving as intended behavior.

File: src/additions/seasonalPerks.json5 [hunk 2]
```diff
@@ -0,0 +1,1116 @@
...
+    points: null,
+    mutuallyExclusiveSeasonalPerkIds: [],
+    effects: [
+      {
+        skillIds: ['Crafting'],
+        intValue: 51,
+        effectId: 'skill_level_preset',
+      },
+      {
+        multiplicator: 0.87,
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/additions/seasonalPerks.json5:70]: **Intentional — kept exactly as the game presents it.** `description` is the verbatim in-game display text; `effects[]` are the raw values the game actually applies. BSG's display strings and underlying values are not always consistent — confirmed for Handyman (text "-50%" vs `craft_time_multiplicator` 0.87), Polydipsia (text "15%" vs `hydration_drain_multiplicator` 1.2), and Average (text/`skill_level_preset` exclude Crafting while `skill_not_growing` includes it).

Both sides are captured faithfully and must **not** be reconciled — "fixing" either side would misrepresent the game. This is now documented in the file header and with inline `NB:` notes on each of the three perks (commit 6a55758). Resolving as intended behavior.

File: src/additions/seasonalPerks.json5 [hunk 3]
```diff
@@ -0,0 +1,1116 @@
...
+    description: '• Limb fracture chance is increased by 25%\n• Falling from heights deals 20% more damage',
+    points: 3,
+    mutuallyExclusiveSeasonalPerkIds: ['69c41632d7a7b2ca660e0c59'],
+    effects: [
+      {
+        multiplicator: 1.25,
+        effectId: 'fracture_chance_multiplicator',
+      },
+      {
+        multiplicator: 1.2,
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/additions/seasonalPerks.json5:158]: **Intentional — kept exactly as the game presents it.** `description` is the verbatim in-game display text; `effects[]` are the raw values the game actually applies. BSG's display strings and underlying values are not always consistent — confirmed for Handyman (text "-50%" vs `craft_time_multiplicator` 0.87), Polydipsia (text "15%" vs `hydration_drain_multiplicator` 1.2), and Average (text/`skill_level_preset` exclude Crafting while `skill_not_growing` includes it).

Both sides are captured faithfully and must **not** be reconciled — "fixing" either side would misrepresent the game. This is now documented in the file header and with inline `NB:` notes on each of the three perks (commit 6a55758). Resolving as intended behavior.

File: src/additions/seasonalPerks.json5 [hunk 4]
```diff
@@ -0,0 +1,1116 @@
...
+        effectId: 'trader_prices_by_trader_multiplicator',
+      },
+    ],
+  },
+  // Average  (points: -12)
+  '69ce64064e0a1c7ca20781d3': {
+    id: '69ce64064e0a1c7ca20781d3',
+    type: 'personal',
+    name: 'Average',
+    description: '• All character skills are set to level 25 but cannot be leveled further (Excluding Crafting skill)',
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/additions/seasonalPerks.json5:764]: **Intentional — kept exactly as the game presents it.** `description` is the verbatim in-game display text; `effects[]` are the raw values the game actually applies. BSG's display strings and underlying values are not always consistent — confirmed for Handyman (text "-50%" vs `craft_time_multiplicator` 0.87), Polydipsia (text "15%" vs `hydration_drain_multiplicator` 1.2), and Average (text/`skill_level_preset` exclude Crafting while `skill_not_growing` includes it).

Both sides are captured faithfully and must **not** be reconciled — "fixing" either side would misrepresent the game. This is now documented in the file header and with inline `NB:` notes on each of the three perks (commit 6a55758). Resolving as intended behavior.

File: src/schemas/seasonal-perk.schema.json [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        "description": "Perk ids that cannot be selected alongside this one.",
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      "effects": {
+        "description": "Effects the perk grants or imposes.",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "additionalProperties": true,
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/schemas/seasonal-perk.schema.json:42]: **Intentionally left open.** Seasonal-perk `effects[]` are heterogeneous BSG effect objects: beyond the enumerated fields, different `effectId`s carry effect-specific keys (`mode`, `traderIds`, `tradeAction`, `bodyPartTypes`, `keyTypes`, `periodUnixSeconds`, `mailTemplateId`, `randomSlotCount`, `appliedRandomEffectCount`, `subEffects`, …). This open shape is deliberate and is documented in `docs/INTEGRATION.md` as `[key: string]: unknown`.

Enforcing `additionalProperties: false` here would require enumerating every current and future BSG effect field and would break as new perks/effects are added. The perk object and `itemFilter` remain strict (`additionalProperties: false`); only the effect object stays open by design. Resolving as intended.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,26 +1,31 @@
...
 - [ ] I have noted the original incorrect value in inline comments
 - [ ] I have included the entity name as a comment above each ID
 - [ ] Field names match tarkov.dev schema exactly (camelCase)
 - [ ] Validation passes locally (`npm run validate`)
 
 ## Related Issues
+
 <!-- Link any related issues -->
+
 Closes #
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: Addressed in 6a55758. The PR template checklist now covers typecheck/tests (`npm run typecheck`, `npm test`) and, for data changes, building + committing the regenerated `dist/overlay.json`, plus a new **Commands Run** section. `docs/CONTRIBUTING.md` now requires `npm run build` for data changes and records the commands run in the PR flow.

File: docs/INTEGRATION.md [hunk 7]
```diff
@@ -242,6 +267,92 @@ function getTaskAdditionsForMode(overlay: Overlay, gameMode: GameMode): TaskAddi
 }
 ```
 
+> **Seasonal mode = `pvp-season`.** BSG's Seasonal Character (EFT patch 1.1.0.0)
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/INTEGRATION.md:270]: Addressed. Every documented `type GameMode` alias now includes `'pvp-season'` — the full fetch example (L554) was updated in cf6e887, the others in 6a55758.

File: docs/INTEGRATION.md [hunk 8]
```diff
@@ -242,6 +267,92 @@ function getTaskAdditionsForMode(overlay: Overlay, gameMode: GameMode): TaskAddi
...
+  type: string; // tier: "common" | "personal"
+  name: string;
+  description?: string;
+  points?: number | null; // perk-point value; null = none
+  mutuallyExclusiveSeasonalPerkIds?: string[];
+  effects?: SeasonalPerkEffect[];
+}
+
+// Only meaningful when the player is on a Seasonal Character.
+const perks = Object.values(overlay.seasonalPerks ?? {});
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/INTEGRATION.md:313]: Addressed in 6a55758. The documented `Overlay` interface now declares optional `seasonalPerks` and `craftsAdd`, and a `Craft` interface is defined in the *Applying Added Crafts* section so the examples compile.

File: src/schemas/craft-additions.schema.json [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+    "craftItem": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["item", "count"],
+      "properties": {
+        "item": {
+          "description": "tarkov.dev item id.",
+          "type": "string"
+        },
+        "count": {
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/schemas/craft-additions.schema.json:63]: Addressed in cf6e887 — `count` is now `type: integer` with `minimum: 1`.

File: src/schemas/craft-additions.schema.json [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+        "minimum": 1
+      },
+      "taskUnlock": {
+        "description": "Id of the task that unlocks this craft (tarkov.dev task id). null on quest-gated crafts whose unlocking task id is not yet known; omit entirely when the craft has no task unlock.",
+        "type": ["string", "null"]
+      },
+      "duration": {
+        "description": "Production time in seconds.",
+        "type": "integer",
+        "minimum": 0
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/schemas/craft-additions.schema.json]: Addressed in cf6e887 — `duration` `minimum` is now `1`.

File: src/schemas/seasonal-perk.schema.json [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+          "required": ["effectId"],
+          "properties": {
+            "effectId": {
+              "description": "BSG effect identifier, e.g. \"pmc_experience_multiplicator\".",
+              "type": "string"
+            },
+            "multiplicator": { "type": "number" },
+            "multiplicatorPrimary": { "type": "number" },
+            "multiplicatorSecondary": { "type": "number" },
+            "intValue": { "type": "number" },
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/schemas/seasonal-perk.schema.json]: Addressed in cf6e887 — `intValue` is now typed `integer`.

File: scripts/wiki-compare/compare.ts [hunk 12]
```diff
@@ -214,8 +187,7 @@ export function compareTasks(
     const apiVerb = getObjectiveVerbKey(apiObj.description ?? '');
     const apiItemRefs = collectObjectiveItems(apiObj);
     const apiFoundInRaid =
-      apiObj.foundInRaid === true ||
-      /found in raid/i.test(apiObj.description ?? '');
+      apiObj.foundInRaid === true || /found in raid/i.test(apiObj.description ?? '');
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/wiki-compare/compare.ts]: Addressed in cf6e887 — the found-in-raid heuristic now excludes the negated `not found in raid` phrasing.

File: scripts/wiki-compare/wiki.ts [hunk 13]
```diff
@@ -269,8 +251,7 @@ export function parseObjectives(
     const mapsFromLinks = mapLinkEntries
       .filter((entry) => !isExcludedMapMention(clean, entry.link))
       .map((entry) => entry.canonical);
-    const mapsFromText =
-      mapsFromLinks.length > 0 ? [] : extractMapsFromText(clean, mapAliasMap);
+    const mapsFromText = mapsFromLinks.length > 0 ? [] : extractMapsFromText(clean, mapAliasMap);
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/wiki-compare/wiki.ts]: Addressed in cf6e887 — text-derived maps are now merged unconditionally, so objectives mixing linked and plain-text map mentions keep both.

File: package.json [hunk 14]
```diff
@@ -15,8 +15,8 @@
     "wiki:compare": "tsx scripts/wiki-compare.ts",
     "monitor": "node monitor/server.js",
     "typecheck": "tsc --noEmit",
-    "format": "prettier --write \"{scripts,src/lib,tests,examples}/**/*.ts\"",
-    "format:check": "prettier --check \"{scripts,src/lib,tests,examples}/**/*.ts\"",
+    "format": "prettier --write \"**/*.{ts,json,json5,md}\"",
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply package.json:18]: Addressed in cf6e887 — `data/` and `eft/` are now in `.prettierignore`.

File: docs/CONTRIBUTING.md [hunk 15]
```diff
@@ -1,233 +1,246 @@
...
+Additions live in `src/additions/` and should include the full object.
+
+## Project Data Layout
+
+- `src/overrides/`: Corrections to existing tarkov.dev entities (tasks, items, traders, hideout).
+- `src/additions/`: New entities not present in tarkov.dev (`tasksAdd`, `editions`, `itemsAdd`, `storyChapters`, `seasonalPerks`, `craftsAdd`).
+- `src/schemas/`: JSON Schemas used by `npm run validate`.
+- `dist/overlay.json`: Generated output from `npm run build` (committed to the repo; regenerate it whenever you change source data).
+  Overrides are keyed by tarkov.dev IDs; additions are keyed by local IDs and
+  appear under their source filenames in the output (`tasksAdd`, `editions`, `itemsAdd`, `storyChapters`, `seasonalPerks`, `craftsAdd`).
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/CONTRIBUTING.md:25]: Addressed in daae634. *Create a Stable ID* now documents that the top-level key format is per-file — local `snake_case` for `tasksAdd`/`editions`, kebab-case for `storyChapters`, and the source tarkov.dev/BSG id for `seasonalPerks`/`craftsAdd` — with `id` always matching the key. tarkov.dev ids are for *referenced* entities only, not top-level keys.

File: docs/CONTRIBUTING.md [hunk 16]
```diff
@@ -1,233 +1,246 @@
...
+- New items → `src/additions/itemsAdd.json5` (schema `item-additions.schema.json`)
+- New seasonal (`pvp-season`) perks → `src/additions/seasonalPerks.json5` (schema `seasonal-perk.schema.json`)
+- New hideout crafts missing from the API → `src/additions/craftsAdd.json5` (schema `craft-additions.schema.json`)
+
+`seasonalPerks` and `craftsAdd` cover `pvp-season` (Seasonal Character) data
+tarkov.dev does not serve. Keep every new data type in `src/additions/` with its
+schema in `src/schemas/`, and register the schema in `SCHEMA_CONFIGS`
+(`src/lib/types.ts`). Prove additions the same way as any other: patch notes
+and/or an in-game screenshot, and use tarkov.dev ids for every referenced
+item/skill/trader/station so names resolve upstream.
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply docs/CONTRIBUTING.md]: Addressed in daae634. Added an explicit **Upstream-id exception**: referenced entities use tarkov.dev ids where available, but a few upstream ids are not in tarkov.dev yet (notably the `Usec*` Seasonal Character faction skills, whose `Bear*` equivalents exist upstream). The correct upstream id is used anyway and resolves once tarkov.dev adds the entity.

**Comment by DysektAI** [comment]
**Review comments folded — status of the latest CodeAnt suggestions (up to d8310b6):**

- ✅ **eft-compare.ts:146 — newest-reference selection**: fixed in `8d82bae`. `findReferenceFile`/`loadEftTasks` now take an optional `mode` and prefer the newest capture matching the requested `--mode`; `requireMatchingReferenceMode` only refuses when no matching capture exists. Callers (eft-compare, eft-audit, eft-wiki-crosscheck) pass the mode through. Regression tests added in `tests/eft-compare.test.ts`.
- ✅ **compare.ts:268 — negated found-in-raid**: fixed in `8d82bae`. The redundant-find filter now excludes `not found in raid` (mirrors the objective-level check from cf6e887). Regression test added.
- ✅ **normalize.ts:99 — skill-level values stripped**: intentionally **not changed**. The strip unifies word-order variants so API/wiki skill objectives match; value differences are caught by the separate count comparison, and the API serves no count for skill objectives, so keeping numbers would only produce false unmatched-objective noise. The skill name remains in the match key.
- ✅ **wiki.ts:193 — verb-count fallback**: fixed in `8d82bae`. The fallback now rejects numbers followed by time/distance units (`minutes`, `seconds`, `hours`, `meters`, `km`) via a negative lookahead, so "for 5 minutes"/"over 40 meters" no longer parse as counts. Regression tests added.
- ✅ **cli.ts:220 — --wiki cache poisoning**: fixed in `8d82bae`. `runSingleTask` no longer writes the wiki cache when an explicit `--wiki` override was supplied.

Validation: `npm run validate`, `npm run typecheck`, 367 tests, `npm run check-overrides` (0 stale / 0 conflict / 0 removed-from-API), and `npm run build` all pass on the merged branch.

**Comment by DysektAI** [comment]
**Review follow-up — disposition of gitar's three remaining suggestions** (latest commit `13d0ec2`):

1. **Edge Case — `mutuallyExclusiveSeasonalPerkIds` cross-references.** Already addressed: `tests/seasonal-perks.test.ts` (commit `8d82bae`) asserts every reference resolves to a defined perk, is mirrored back symmetrically, and every `id` matches its object key. The schema keeps the field as a string array by design — JSON Schema cannot express cross-file referential integrity, so a test is the right enforcement point.

2. **Quality — perk `effects[]` allow arbitrary properties.** Intentional, not a defect. BSG effect objects are heterogeneous (effect-specific keys like `mode`, `traderIds`, `tradeAction`, `bodyPartTypes`, `periodUnixSeconds`, `subEffects`, …), so `additionalProperties: true` is deliberate and documented in `docs/INTEGRATION.md`; the perk object and `itemFilter` remain strict.

3. **Bug — text-only maps dropped when an objective has any linked map** (`scripts/wiki-compare/wiki.ts:276-281`). Valid observation, but the gate is pre-existing in `main` (since #227) and PR #257 does not change that line in its net diff. Out of scope for this PR → tracked in #266.

No code changes resulted from these three — the branch is already green (all CI checks pass, 0 unresolved review threads).

File: scripts/wiki-compare/cli.ts [hunk 17]
```diff
@@ -244,12 +244,17 @@ export async function runSingleTask(
...
+    // task's regular wiki page; caching it under the task id would poison the
+    // cache for later plain runs (they would compare against the wrong page).
+    if (!options.wiki) {
+      saveWikiCache(
+        task.id,
+        wikiResponse.title,
+        wikiResponse.wikitext,
+        wikiResponse.lastRevision
+      );
+    }
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/wiki-compare/cli.ts:227]: Fixed in a9eef7d: the cache read is now gated on `!options.wiki`, so an explicit `--wiki` page is always fetched even when a task cache exists (previously the write guard was the only protection and a stale cached page would be compared instead).

File: src/lib/divergence-validator.ts [hunk 18]
```diff
@@ -61,6 +61,34 @@ export interface DivergenceModeContext {
...
+function overrideFieldValue(entry: TaskOverride | undefined, field: string): unknown {
+  if (!entry) return undefined;
+  const objectiveId = objectiveFieldId(field);
+  if (objectiveId) {
+    const objectives = (entry as unknown as Record<string, unknown>).objectives as
+      Record<string, { count?: unknown }> | undefined;
+    return objectives?.[objectiveId]?.count;
+  }
+  return (entry as unknown as Record<string, unknown>)[field];
+}
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/lib/divergence-validator.ts:88]: Fixed in a9eef7d: `overrideFieldValue` now reads `entry.objectives` through the typed `TaskOverride` field instead of the double cast.

File: src/overrides/modes/pve/tasks.json5 [hunk 19]
```diff
@@ -22,106 +20,111 @@
...
-  // Verified 2026-08-03 against a local EFT PvE client capture (2026-06-30) and
-  // tarkov.dev; recorded in src/divergences/tasks.json5.
-  //
-  // The `Was:` values are the ORIGINAL regular-mode figures observed when each
-  // override was authored, retained as historical evidence.
-  // ===========================================================================
-
-  // --- Converged: EFT reference, wiki and upstream now agree in both modes ---
+  // so the field no longer applies to quests. (Confirmed against the 1.1.0.0
+  // PvE capture: none of the formerly-divergent quests carry a Level condition.)
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/overrides/modes/pve/tasks.json5]: Fixed in a9eef7d: the `minPlayerLevel` removal note now names Collector as the surviving 1.1.0.0 exception, and Counteraction / Kings of the Rooftops were moved under the CONVERGED banner (their `status` fields were already `converged`).

File: scripts/eft-compare.ts [hunk 20]
```diff
@@ -112,18 +112,100 @@ function asNumber(value: unknown): number | undefined {
...
+function modeFromReferenceFile(file: string): GameMode | null | 'unusable' {
+  try {
+    const raw = JSON.parse(readFileSync(file, 'utf-8')) as {
+      request?: { url?: string };
+    };
+    return modeFromRequestUrl(raw?.request?.url);
+  } catch {
+    return 'unusable';
+  }
+}
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/eft-compare.ts:223]: Reasonable micro-optimization, but this is a one-shot CLI audit over a handful of captures, and the hot path (the sort comparator) was already fixed in 4e0b4cc to precompute ranks. Adding a module-level envelope cache introduces mutable state for negligible gain in this tool, so I'm leaving it as-is.

File: scripts/eft-compare.ts [hunk 21]
```diff
@@ -112,18 +112,100 @@ function asNumber(value: unknown): number | undefined {
...
+      // An unparseable capture cannot be used for comparison, so skip it; a
+      // valid capture with an inconclusive URL stays plausible (callers then
+      // trust --mode).
+      if (candidateMode === null || candidateMode === mode) {
+        return candidate;
+      }
+    }
+  }
+  return ranked[0];
+}
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/eft-compare.ts:183]: No wrong data reaches any report today: all three CLI entry points call `requireMatchingReferenceMode` immediately after `findReferenceFile`, which aborts on a mode mismatch. The `ranked[0]` fallback is only reachable when every candidate is provably another mode, which that guard rejects first. Changing the exported function's contract for this hypothetical future-misuse case is out of scope for this data PR.

File: scripts/eft-compare.ts [hunk 22]
```diff
@@ -492,7 +591,7 @@ async function main(): Promise<void> {
     const opts = parseModeArgs(process.argv.slice(2), ['--descriptions']);
 
     printProgress(`Parsing quest reference file from ${opts.eftDir}...`);
-    const refFile = findReferenceFile(opts.eftDir);
+    const refFile = findReferenceFile(opts.eftDir, opts.mode);
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/eft-compare.ts:616]: Not a bug: the `eft:*` tools compare a local quest reference capture against tarkov.dev, and those captures exist only for `regular` and `pve` (no `pvp-season` quest reference is captured). Accepting `--mode pvp-season` would select a nonexistent reference, so the CLI intentionally limits `--mode` to the modes with reference data. The `pvp-season` test case correctly exercises the guard's rejection path, not CLI selection.

File: src/overrides/modes/regular/tasks.json5 [hunk 23]
```diff
@@ -27,6 +27,52 @@
...
+
+  // Power of Persuasion - PvP: 20 PMCs with MP5SD on Streets/Ground Zero/
+  // Interchange (wiki); PvE: 5 PMCs with MP5SD on Woods/Ground Zero/Interchange/
+  // Customs (captures). tarkov.dev serves the PvE variant in both modes.
+  // Proof: https://escapefromtarkov.fandom.com/wiki/Power_of_Persuasion
+  '63a5cf262964a7488f5243ce': {
+    objectives: {
+      '63a5cf262964a7488f5243d1': {
+        description: 'Eliminate 20 PMC operatives while using an MP5SD on Streets of Tarkov, Ground Zero or Interchange', // Was (PvE): "Eliminate PMC operatives with an MP5SD on Woods, Ground Zero, Interchange, or Customs"
+        count: 20, // Was: 5 (tarkov.dev mirrors PvE count)
```
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/overrides/modes/regular/tasks.json5]: Descriptions here are verbatim source text — the wiki's PvP wording — which includes the count. The structured `count` field is separate and is what consumers render alongside the description. The override at lines 12-15 omits the count only because that wiki text doesn't include it; stripping the count from these two would misrepresent the source, so I'm keeping the text faithful.

File: scripts/eft-compare.ts (hunk 20 above)
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/eft-compare.ts:223]: Follow-up: addressed in a40c8b5. Added `readReferenceEnvelope` with a per-file cache so `captureTimestamp` and `modeFromReferenceFile` parse each dump once across `findReferenceFile` and `requireMatchingReferenceMode`.

File: scripts/eft-compare.ts (hunk 21 above)
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/eft-compare.ts:183]: Follow-up: addressed in a40c8b5. `findReferenceFile` now throws when `--mode` has no matching or inconclusive capture instead of returning a cross-mode file; `loadEftTasks` degrades it to null. Regression coverage added in `tests/eft-compare.test.ts`.

File: scripts/eft-compare.ts (hunk 22 above)
[the remark this answers is not in the document]
Comment (DysektAI) [reply scripts/eft-compare.ts:616]: Follow-up: addressed in a40c8b5. Behavior is unchanged (there is no pvp-season quest reference, so `--mode` stays limited to regular/pve), but I added a comment documenting why the restriction differs from `SUPPORTED_GAME_MODES`.

File: src/overrides/modes/regular/tasks.json5 (hunk 23 above)
[the remark this answers is not in the document]
Comment (DysektAI) [reply src/overrides/modes/regular/tasks.json5]: Follow-up: verified against the wiki and addressed in a40c8b5. The wiki does embed the count ("Eliminate 20 PMCs…", "Eliminate any 50 targets…"), but tarkov.dev's convention and the Vacate the Premises override both keep counts out of the description, so I removed "20"/"50" from the Power of Persuasion and Easy-Breezy descriptions and rebuilt dist/overlay.json.
